submarine-workbench/workbench-web/src/WEB-INF/web.xml

```diff
@@ -30,11 +30,6 @@
       <param-name>jersey.config.server.provider.packages</param-name>
       <param-value>io.swagger.v3.jaxrs2.integration.resources,org.apache.submarine.server.rest</param-value>
     </init-param>
-    <init-param>
-      <param-name>openApi.configuration.resourceClasses</param-name>
-      <param-value>org.apache.submarine.server.rest.ExperimentRestApi, 
-      org.apache.submarine.server.rest.ExperimentTemplateRestApi</param-value>
-    </init-param>
```

Comment:
Why do you delete these codes?

Reply:
I set `openApi.configuration.resourceClasses` in `Bootstrap.java`